### DISCUSSION TITLE
Calendar event status

### DISCUSSION THREAD

**Discussion by sistemasnow** [YetiForceCompany/YetiForceCRM#15401] [How to]
Dear friends:

Is there any way to change the **completed status** in a calendar event? 

Example: you marked (by mistake) an event as completed and it dissapears from calendar views, but we need to mark it again as not completed (or similar) to get it back to calendar. 

The best option would be to change that would be to mass editing, but the field status does not appear in mass edit windows. 

Any idea? 

YetiForce version: 6.1.0

Thanks in advance.

**Comment by rskrzypczak** [comment]
I do not see such a possibility from GUI.
You need to create new entries

**Reply by sistemasnow** [reply]
Thanks, Radoslaw for your answer.

Best regards!

Jorge G.

**Comment by diderich** [comment]
You can do that by following these steps (note that I do not know if changing the type of the status field has negative implications to the system - any input from the developers):
1. Change the "status" field type to "Field visible everywhere" in "Standard Modules ->Edit Fields"
![step1](https://user-images.githubusercontent.com/10230144/140334039-4aa3f018-4f44-444f-b452-1fdd79dbc341.jpg)

2. Go to the calendar page and select "Record list"
![step2](https://user-images.githubusercontent.com/10230144/140332419-62a252a0-c946-4b60-a44d-dc94a5ed6f58.jpg)

3. Select meeting of which you want to change the "status"
![step3](https://user-images.githubusercontent.com/10230144/140332537-ae576ae0-3ba4-48a9-808f-0b5bac88bd15.jpg)

 4. Edit the entry
![step4](https://user-images.githubusercontent.com/10230144/140332609-06e9ef48-7680-4b9e-a457-76721defdc6f.jpg)

5. Now you can edit the "status". But note that when saving a meeting in the past, you cannot reset it to "planned" or "pending"
![tep5](https://user-images.githubusercontent.com/10230144/140333011-a160aa10-4d70-4205-969a-5348e1a474ca.jpg)

**Reply by sistemasnow** [reply]
Hi, diderich:

Thanks so much for your answer.

A few things:

- The "status" field we would like to change is the "activitystatus" one (not Contact option).

![Anotación 2021-11-04 160925](https://user-images.githubusercontent.com/83352572/140343734-d06f9c10-79c5-4052-86ad-6c2bf8ab3b99.png)
- We are unable to see the "find visible everywhere" check box into the Edit field option. 
 
![Anotación 2021-11-04 161000](https://user-images.githubusercontent.com/83352572/140344445-a0f6f42b-12c7-402c-ad87-89526de146d5.png)

So we are stuck at this point. 

And, as you mentioned it, we need to change that field into "planned" or "pending" in order to make it reappear.

So I´m afraid that Radoslaw is right and we´ll have to create the events again.

Thanks so much. 

Jorge G.

**Reply by diderich** [reply]
That's what my example shows!?
